MAESTRO/Snakemake/multi_scATAC/rules/sc_atac_peak_count.smk

```diff
@@ -0,0 +1,57 @@
+_countpeak_threads = 4
+
+rule scatac_countpeak:
+    input:
+        finalpeak = "Result/Analysis/{sample}/{sample}_final_peaks.bed",
+        validbarcode = "Result/QC/{sample}/{sample}_scATAC_validcells.txt",
+        frag = "Result/minimap2/{sample}/fragments_corrected_dedup_count.tsv"
+    output:
+        counts = "Result/Analysis/{sample}/{sample}_peak_count.h5"
+    params:
+        species = config["species"],
+        outdir = "Result/Analysis/{sample}",
+        outpre = "{sample}"
+    threads:
+        _countpeak_threads
+    benchmark:
+        "Result/Benchmark/{sample}_PeakCount.benchmark"
+    shell:
+        """
+        MAESTRO scatac-peakcount --peak {input.finalpeak} --fragment {input.frag} --barcode {input.validbarcode} \
```

Comment:
should we turn on --binary for multi-sample as well? I see you turned it on for single sample. I am still investigating which way is better. before we get a conclusion, let's make the single-sample and mutli-sample workflow consistent.